src/test/java/seedu/recipe/logic/commands/DeleteCommandTest.java

```diff
@@ -78,24 +82,34 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
 
     @Test
     public void equals() {
-        DeleteCommand deleteFirstCommand = new DeleteCommand(INDEX_FIRST_PERSON);
-        DeleteCommand deleteSecondCommand = new DeleteCommand(INDEX_SECOND_PERSON);
+        DeleteCommand deleteFirstCommand = new DeleteCommand(INDEX_FIRST_RECIPE);
+        DeleteCommand deleteSecondCommand = new DeleteCommand(INDEX_SECOND_RECIPE);
 
         // same object -> returns true
-        assertTrue(deleteFirstCommand.equals(deleteFirstCommand));
+        assertEquals(deleteFirstCommand, deleteFirstCommand);
 
         // same values -> returns true
-        DeleteCommand deleteFirstCommandCopy = new DeleteCommand(INDEX_FIRST_PERSON);
-        assertTrue(deleteFirstCommand.equals(deleteFirstCommandCopy));
+        DeleteCommand deleteFirstCommandCopy = new DeleteCommand(INDEX_FIRST_RECIPE);
+        assertEquals(deleteFirstCommand, deleteFirstCommandCopy);
 
         // different types -> returns false
-        assertFalse(deleteFirstCommand.equals(1));
+        assertNotEquals(1, deleteFirstCommand);
 
         // null -> returns false
-        assertFalse(deleteFirstCommand.equals(null));
+        assertNotEquals(null, deleteFirstCommand);
 
         // different recipe -> returns false
-        assertFalse(deleteFirstCommand.equals(deleteSecondCommand));
+        assertNotEquals(deleteFirstCommand, deleteSecondCommand);
+    }
+
+    @Test
+    public void test_executeNullModel() {
+        assertThrows(NullPointerException.class, () -> new DeleteCommand(INDEX_FIRST_RECIPE).execute(null));
+    }
+
+    @Test
+    public void test_nullIndexConstructor() {
```

Comment:
names for these 2 tests doesn't follow coding standard

Reply:
fixed